app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -433,6 +437,36 @@ class AdministratorControlsFragmentTest {
     }
   }
 
+  @Test
+  fun testFragmentArguments_afterCreation_areCorrect() {
```

Comment:
Suggest name format for all the tests in PR.
```suggestion
  fun testFragment_argumentsAreCorrect() {
```

Reply:
Done